### ISSUE TITLE
build issue in VS2017 for windows 10

### ISSUE THREAD

**Issue by asheesh1202** [valhalla/valhalla#2129]
I am building valhalla on windows 10. I am able to generate valhalla.sln file using cmake. When I open it in VS2017 and try to build it I get following error.

`
Error	C1083	Cannot open include file: 'zlib.h': No such file or directory	valhalla-mjolnir	e:\open_steet_map\valhalla\src\mjolnir\osmpbfparser.cc	38	
`

I get same file not found error for boost, zlib, lua and all files inside valhalla/proto/.

**Comment by asheesh1202** [comment]
same error pops for lua library files also

**Comment by PureW** [comment]
You are missing some dependencies. Take a look at the Ubuntu instructions at https://github.com/valhalla/valhalla#building-from-source and make sure you have the correct dependencies installed.

**Comment by kevinkreiser** [comment]
additionally you can have a look at the CI we use for building windows: https://github.com/valhalla/valhalla/blob/master/.azure-pipelines.yml#L40

**Comment by asheesh1202** [comment]
@PureW the link mentioned is for Ubuntu setup which I found is quite different from Windows Setup. 
Building Valhalla in windows is becoming painful for me.

However, I found that while building baldr, Midgard and Loki, I error was automatically removed but when All_Valhalla_Products are give for building I get this error. So this might be due to some build problem that is in particular to my system and nothing is required to be done by valhalla team. 

Thanks for the help
